src/components/Tabs/Tabs.tsx

```diff
@@ -12,15 +13,68 @@ interface ITabsProps {
 }
 
 export const Tabs = ({ children }: ITabsProps) => {
+  const [showScrollButton, setShowScrollButton] = useState(false);
+  const [disableScrollLeft, setDisableScrollLeft] = useState(true);
+  const [disableScrollRight, setDisableScrollRight] = useState(false);
+  const tabContent = useRef<HTMLUListElement>(null);
+
+  const onResize = useCallback(() => {
+    tabContent.current && setShowScrollButton(tabContent.current.scrollWidth > window.innerWidth);
+  }, [tabContent]);
+
+  const scrollLeft = () => {
+    if (tabContent.current) {
+      setDisableScrollRight(false);
+      setDisableScrollLeft(tabContent.current.scrollLeft - 150 <= 0);
+      tabContent.current.scrollLeft = tabContent.current.scrollLeft ? tabContent.current.scrollLeft - 150 : 0;
```

Comment:
`150` should be replaced by some constant, for example `SCROLL_SIZE`, then it should be reused everywhere